libraries/datastore/src/main/java/no/dcat/datastore/domain/dcat/client/LoadLocations.java

```diff
@@ -65,7 +65,13 @@ public void addLocationsToThemes(Model model, Map<String, SkosCode> existingLoca
         ResIterator resIterator = model.listResourcesWithProperty(DCTerms.spatial);
         resIterator.forEachRemaining(datasetResource -> {
             datasetResource.listProperties(DCTerms.spatial).forEachRemaining(spatial -> {
-                String uri = AbstractBuilder.removeDefaultBaseUri(model, spatial.getObject().asResource().getURI());
+
+                String uri = null;
+                if(spatial.getObject().isResource()) {
+                    uri = AbstractBuilder.removeDefaultBaseUri(model, spatial.getObject().asResource().getURI());
+                } else {
+                    uri = spatial.getLiteral().getString();
```

Comment:
Antar denne linjen er workarounden.

Kan det være en ide å legge inn en kommentar for å understreke det ?